src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -85,6 +97,22 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'markdownDescription': localize('decorations.colors', "Controls whether editor file decorations should use colors."),
 				'default': true
 			},
+			[CustomEditorLabelService.SETTING_ID_ENABLED]: {
+				'type': 'boolean',
+				'markdownDescription': localize('workbench.editor.label.enabled', "Controls whether the custom workbench editor labels should be applied."),
+				'default': true,
```

Comment:
Should this be enabled by default if we have no patterns by default?

Reply:
The idea was that the user can add his pattern and it will automatically work without having to find this setting. The goal of this setting is to be able to easily turn off the applied patterns without having to delete them.